hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -141,6 +141,8 @@
     boolean getBoolean(@Nonnull String fieldName);
 
     /**
+     * Supported only for {@link Portable}. Not applicable for {@link com.hazelcast.config.CompactSerializationConfig Compact}
+     *
```

Comment:
We need to put a similar comment to `char[] getArrayOfChar(@Nonnull String fieldName);`